airflow/cli/cli_parser.py

```diff
@@ -977,6 +977,17 @@ class GroupCommand(NamedTuple):
             ARG_SAVE_DAGRUN,
         ),
     ),
+    ActionCommand(
+        name='reserialize',
+        help="Reserialize all DAGs by parsing the DagBag files",
+        description=(
+            "Drop all serialized dags from the metadata DB. This will cause all DAGs to be reserialized "
+            "from the DagBag folder. This can be helpful if your serialized DAGs get out of sync with the "
+            "version of Airflow that you are running."
+        ),
+        func=lazy_load_command('airflow.cli.commands.dag_command.dag_reserialize'),
```

Comment:
1. I thought I did nest it there? That was my intention. Am I misreading something?
2. I was thinking that it would be better to keep all the dag parsing logs together, but that's not a very strongly held opinion.
3. Will do.

Reply:
(2) As the name is `reserialize` -- it will only reserialize if the scheduler is running. If the scheduler is not running for whatever reason, the users will have a wrong impression that `reserialize` does actually deserialize instead of just "clear"/delete